traffic_portal/app/src/common/modules/form/_form.scss

```diff
@@ -370,3 +370,10 @@ input:checked + .slider::before {
 		text-shadow: none;
 	}
 }
+
+.checkbox-scroll-container {
+	max-height: 3.5em;
+	overflow-y: auto;
+	display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
```

Comment:
I know this grid was my idea, but thinking more about it, this looks fine on desktops, but it'll be weird on mobile. The rest of the form is responsive to screen size, but this is always 8 columns, even when that doesn't fit. So instead of a grid, it's probably best to just flex. Given this HTML:

```html
<div class="checkbox-scroll-container">
    <div ng-repeat="capability in requiredCapabilities">
        <input
            type="checkbox"
            id="capability"
            name="capability"
            ng-checked="selectedCapabilities[capability.name]"
        />
        <label>
            <a ng-href="/#!/server-capabilities/edit?name={{capability.name}}" target="_blank"> {{capability.name}}</a>
        </label>
    </div>
</div>
```

I wrote this SCSS
```scss
.checkbox-scroll-container {
	max-height: 3.5em;
	overflow-y: auto;
	display: flex;
	flex-wrap: wrap;
	justify-content: space-between;
	align-content: space-between;

	div {
		min-width: 5em;
		width: fit-content;
	}
}
```
which does pretty much exactly what we want. It'll space the controls evenly in the space it has and if it runs out it'll just wrap to the next line. It doesn't guarantee any number of capabilities on a single line, because the size of the control depends on the size of the name of the capability. Should work fine on mobile as well as it does on desktops.